### PR TITLE
Update Readme and MD to use ipynb extension for VS Code links

### PR DESCRIPTION
**Pull request by JakeRadMSFT**
1. Updating readme to use the ipynb version of the VS Code links.
2. Update MD comments to use ipynb version of VS Code links for Next/Previous links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,21 +15,21 @@ Download or clone this repo and open the `csharp-101` folder in VS Code to get s
...
+7  | Branches (if)                 | [07 Notebook](https://tinyurl.com/csharp101-notebook07-ipynb) | [07 Video](https://www.youtube.com/watch?v=y4OTe8LSokg&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=8) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
+8  | What Are Loops?               | [08 Notebook](https://tinyurl.com/csharp101-notebook08-ipynb) | [08 Video](https://www.youtube.com/watch?v=z31m5Up_gSQ&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=10) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
+9  | Combining Branches and Loops  | [09 Notebook](https://tinyurl.com/csharp101-notebook09-ipynb) | [09 Video](https://www.youtube.com/watch?v=qK7tUpaOXi8&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=11) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
```
Comment (JakeRadMSFT) [inline README.md:26]: Notebook 9 Doesn't work

Comment (JakeRadMSFT) [reply README.md:26]: @jamesmontemagno the url used by the tinyurl is wrong for 2 and 9.

Comment (JakeRadMSFT) [reply README.md:26]: More context:
https://github.com/dotnet/interactive/issues/1861#issuecomment-1027196647

File: csharp-101/shorturl_bulk.csv [hunk 2]
```diff
@@ -0,0 +1,16 @@
+URL,Domain,Alias
```
Comment (JakeRadMSFT) [inline csharp-101/shorturl_bulk.csv:1]: @katiesavage  @jamesmontemagno - this is the file used to bulk-add the tiny urls.

**Review by jamesmontemagno — APPROVED** [review]

**Pull request merged**